[2021-09-27 14:10:56,286] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-09-26T00:00:00+00:00 [queued]>
[2021-09-27 14:10:56,315] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-09-26T00:00:00+00:00 [queued]>
[2021-09-27 14:10:56,316] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:10:56,318] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 14:10:56,319] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 14:10:56,330] {taskinstance.py:1115} INFO - Executing <Task(BashOperator): also_run_this> on 2021-09-26T00:00:00+00:00
[2021-09-27 14:10:56,335] {standard_task_runner.py:52} INFO - Started process 2828 to run task
[2021-09-27 14:10:56,342] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', '2021-09-26T00:00:00+00:00', '--job-id', '48', '--pool', 'default_pool', '--raw', '--subdir', '/home/***/.local/lib/python3.6/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpsu1wvyz_', '--error-file', '/tmp/tmp10gd53f9']
[2021-09-27 14:10:56,345] {standard_task_runner.py:77} INFO - Job 48: Subtask also_run_this
[2021-09-27 14:10:56,421] {logging_mixin.py:109} INFO - Running <TaskInstance: example_bash_operator.also_run_this 2021-09-26T00:00:00+00:00 [running]> on host 6c9f6f41e1ed
[2021-09-27 14:10:56,503] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-09-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-26T00:00:00+00:00
[2021-09-27 14:10:56,506] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-27 14:10:56,507] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2021-09-26T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-09-26 00:00:00+00:00: scheduled__2021-09-26T00:00:00+00:00, externally triggered: False>"']
[2021-09-27 14:10:56,519] {subprocess.py:74} INFO - Output:
[2021-09-27 14:10:56,521] {subprocess.py:78} INFO - run_id=scheduled__2021-09-26T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-09-26 00:00:00+00:00: scheduled__2021-09-26T00:00:00+00:00, externally triggered: False>
[2021-09-27 14:10:56,522] {subprocess.py:82} INFO - Command exited with return code 0
[2021-09-27 14:10:56,560] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20210926T000000, start_date=20210927T141056, end_date=20210927T141056
[2021-09-27 14:10:56,601] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 14:10:56,640] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
